component/authenticator/pom.xml

```diff
@@ -127,6 +132,7 @@
                             org.wso2.carbon.extension.identity.helper.*;
                             version="${identity.extension.utils.import.version.range}"
                         </Import-Package>
+                        <Import-Package>*;resolution:=optional</Import-Package>
```

Comment:
* import is dangerous. You need to import the required package with version range